build-tools/build.sh

```diff
@@ -0,0 +1,223 @@
+#!/bin/bash
+script_start_time=$SECONDS
+
+# ----------------
+# VERIFY INPUTS
+# ----------------
+
+# Check if the user has provided a library folder name if copy_readme_and_license_to_include: is true
+if [[ "$INPUT_COPY_README_AND_LICENSE_TO_INCLUDE" == "true" && -z "$INPUT_LIB_FOLDER_NAME" ]]; then
+    echo "You must provide a library folder name if copy_readme_and_license_to_include is true" >&2
+    echo "You must provide a library folder name if copy_readme_and_license_to_include is true"
+    # Add to Workflow summary
+    echo "# ::error::You must provide a library folder name if copy_readme_and_license_to_include is true" >>$GITHUB_STEP_SUMMARY
+    exit 102502 # This is the string "pros-build" turned into int values, added together, and then multiplied by 10 plus the error code at the end. This is to hopefully avoid conflicts with other error codes.
```

Comment:
points for creativity? afaik exit codes are modulo-ed by 255, but I could be wrong

Reply:
Are they? Idk. I was bored as you may be able to tell.

Reply:
oh damn there is a max of 255 :(